db/queries.js

```diff
@@ -1,6 +1,10 @@
 const Promise = require('bluebird');
 const knex = require('./knex');
-const { isValidDateTime } = require('../lib/utility');
+const _ = require('lodash');
```

Comment:
Perhaps destructure out just the chunk function here, since it seems to be the only one you're using